DoSomething/rogue
# Hooks up S3 to save reportback item photo

#### What's this PR do?
- Adds `file_url` column to the `reportback_items` table to store the URL where the reportback item image is located on S3. 
- Adds functionality to save reportback item photo in S3 when reportback is received from Phoenix.
- Adds s3 testing to `testCreatingNewReportback` test.

#### How should this be reviewed?
- Run `phpunit` and all tests should still pass. 
- Manually, post to `http://rogue.dev:8000/api/v1/reportbacks` in Postman with below as reference:
![screen shot 2016-08-25 at 3 12 10 pm](https://cloud.githubusercontent.com/assets/9019452/17982561/56376836-6ad6-11e6-95b8-982df9b73215.png)
- Check S3 and you should see images uploaded to correct bucket. Works with both file uploads and base64 strings. 
- Database should also reflect the url of where these images live. 

#### Any background context you want to provide?
Follows logic used to [upload user profile photos to S3 in Northstar](https://github.com/DoSomething/northstar/blob/dev/app/Services/AWS.php).

#### Relevant tickets
Fixes #26 

#### Checklist
